### PR TITLE
Move custom components to Semantic

### PR DESCRIPTION
**Pull request by FinnIckler**
This moves the custom UMD Styling to Semantic Components while keeping the overall Layout and Design.

Components Done:
- Competitions Box
- Registration Requirements
- Permissions Message (was already a semantic component, just removed the width and made for flexible)
- Details Box on the bottom of Home
- Event Step (I think we are mostly fine here)
- Information Section
- Processing

### CODE REVIEW

**Comment by FinnIckler** [comment]
@thewca-bot deploy staging

**Comment by FinnIckler** [comment]
@thewca-bot deploy staging

File: Frontend/src/pages/register/components/RegistrationRequirements.jsx [hunk 1]
```diff
@@ -1,64 +1,166 @@
...
+            index={0}
+            onClick={handleClick}
+          >
+            <UiIcon name="dropdown" />
+            WCA Account Required
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 0}>
+            <p>
+              You need a WCA Account to register, click{' '}
+              <a href="/users/sign_up">here</a> to create One
```
Comment (kr-matthews) [inline Frontend/src/pages/register/components/RegistrationRequirements.jsx:38]:
```suggestion
              <a href="/users/sign_up">here</a> to create one.
```

File: Frontend/src/pages/register/components/RegistrationRequirements.jsx [hunk 2]
```diff
@@ -1,64 +1,166 @@
...
+            index={1}
+            onClick={handleClick}
+          >
+            <UiIcon name="dropdown" />
+            {competitionInfo.competitor_limit} person Competitor Limit
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 1}>
+            <p>
+              Once the competitor Limit has been reached you will be put onto
+              the waiting list
```
Comment (kr-matthews) [inline Frontend/src/pages/register/components/RegistrationRequirements.jsx:52]:
```suggestion
              the waiting list.
```

File: Frontend/src/pages/register/components/RegistrationRequirements.jsx [hunk 3]
```diff
@@ -1,64 +1,166 @@
...
+                  competitionInfo.refund_policy_limit_date ??
+                    competitionInfo.start_date
+                ).format('ll')}`}
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 2}>
+            <p>
+              You will get a {competitionInfo.refund_policy_percent}% refund
+              before this date.
+            </p>
+          </Accordion.Content>
```
Comment (kr-matthews) [inline Frontend/src/pages/register/components/RegistrationRequirements.jsx:75]: What if it's the 'no refunds' case?

File: Frontend/src/pages/register/components/RegistrationRequirements.jsx [hunk 4]
```diff
@@ -1,64 +1,166 @@
...
                 competitionInfo.end_date
             ).format('ll')}
-            <UiIcon name="circle info" />
-          </span>
-        }
-      />
-    </div>
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 3}>
+            <p>You can edit your registration until this date</p>
```
Comment (kr-matthews) [inline Frontend/src/pages/register/components/RegistrationRequirements.jsx:89]:
```suggestion
            <p>You can edit your registration until this date.</p>
```

**Comment by FinnIckler** [comment]
@thewca-bot deploy staging

**Comment by FinnIckler** [comment]
@thewca-bot deploy staging

**Pull request merged**